package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "dependencies": {
+    "bcrypt": "^5.1.0",
+    "by": "^0.2.3",
+    "cors": "^2.8.5",
+    "dotenv": "^16.0.3",
+    "express": "^4.18.2",
+    "nodemon": "^2.0.20",
+    "passport": "^0.6.0",
+    "passport-google-oauth": "^2.0.0",
+    "winston": "^3.8.2"
+  },
+  "name": "team-fintech-backend",
+  "description": "The Backend for our Fintech Produt",
+  "version": "1.0.0",
+  "main": "index.js",
+  "devDependencies": {},
+  "scripts": {
+    "start": "node ./app.js",
+    "dev": "nodemon ./app.js",
```

Comment:
entry file is not app.js??  It is located inside an src directory just like the folder structure and it is server.js. located in ``` src/server.js ```